druid-shell/src/mac/mod.rs

```diff
@@ -362,24 +362,19 @@ extern "C" fn scroll_wheel(this: &mut Object, _: Sel, nsevent: id) {
         let view_state: *mut c_void = *this.get_ivar("viewState");
         let view_state = &mut *(view_state as *mut ViewState);
         let (dx, dy) = {
-            let dx = nsevent.scrollingDeltaX() as i32;
-            let dy = -nsevent.scrollingDeltaY() as i32;
+            let dx = -nsevent.scrollingDeltaX() as f64;
+            let dy = -nsevent.scrollingDeltaY() as f64;
             if nsevent.hasPreciseScrollingDeltas() == cocoa::base::YES {
                 (dx, dy)
             } else {
-                (dx * 32, dy * 32)
+                (dx * 32.0, dy * 32.0)
```

Comment:
What's going on here?

Reply:
This seems to be the traditional constant for external mouse wheel vs trackpad. See https://github.com/makepad/makepad/blob/master/render/src/cx_cocoa.rs#L316 for the corresponding makepad version. Also note that winit keeps the distinction (using LineDelta for an ext mouse wheel, PixelDelta for the trackpad). As you suggest, we might come back to this.